src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -95,6 +96,9 @@ const SpendingByCFDAContainer = ({ onRedirectModalClick }) => {
                 order
             }
         };
+        if (activeTab !== 'all') {
+            params.filter.award_type_codes = awardTypeGroups[activeTab];
```

Comment:
so if the active tab is not all, we dont provide this property in the request object?

Reply:
Correct, the endpoint defaults to all financial assistance types if `award_type_codes` is not provided.